jenkins-integration/jenkins-jobs/common/scripts/job-steps/initialize_ruby_env.sh

```diff
@@ -4,10 +4,10 @@ echo ""
 echo "Setting Up Ruby Environment"
 echo ""
 source /usr/local/rvm/scripts/rvm
-rvm use 2.1.6 || exit 1
+rvm use 2.2.5 || exit 1
```

Comment:
This seems fine but now that we have this requirement, I think we'll need to do the following before it's safe to merge this:

1. Create a new release of the puppetlabs-puppetserver_perf_driver (which is really just an update to the CHANGELOG, plus the creation of a tag on github)
2. File a PR against puppetlabs-modules that bumps the Puppetfile up to that version.